src/components/UserEventsTable.js

```diff
@@ -4,47 +4,169 @@ import { Table, Segment, Header, Grid } from "semantic-ui-react";
 import moment from "moment";
 
 function UserEventsTable({ user }) {
+  const reversedEvents = [];
+  for (let i = user.events.length - 1; i >= 0; i--) {
+    reversedEvents.push(user.events[i])
+  }
+
+  var GBMCount = 0;
+  var workshopCount = 0;
+  var cabinetMeetingCount = 0;
+  var miscCount = 0;
+  
+  {user && user.events.map(event => (
```

Comment:
Why are all of these returing false? Just make them into a for loop as well and save space.

Maybe just make gbm, workshop, cabinetMeeting and misc arrays and map those in the react code?

Also, instead of using user.events, just use reversedEvents.map()